diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/DataClassesRuleWarnTest.kt

```diff
@@ -25,7 +26,22 @@ class DataClassesRuleWarnTest : LintTestBase(::DataClassesRule) {
     }
 
     @Test
-    @Tag("USE_DATA_CLASS")
+    @Tag(USE_DATA_CLASS)
+    fun `should trigger - dont forget to consider this class in fix`() {
+        lintMethod(
+                """
+                    |class Test {
+                    |   var a: Int = 0
+                    |          get() = field
+                    |          set(value: Int) { field = value}
```

Comment:
I think we should also check if these accessors are trivial. If user has some other logic there, it might be not a very good idea to suggest data class conversion. Maybe, check if accessor's body contains only single expression?